### ISSUE TITLE
IL Tools: Installing with dotnetcore

### ISSUE THREAD

**Issue by mletterle** [dotnet/coreclr#10597]
After installing `dotnetcore` via the instructions at https://www.microsoft.com/net/core there is no way to install the il tools. The default `dotnet nuget` command has no `install` command. 

After using `apt-get install nuget` to get the stand-alone nuget-cli, I am able to `nuget install Microsoft.NETCore.ILAsm`, which doesn't appear to pull down any binaries.

Using `nuget install runtime.ubuntu.16.04-x64.Microsoft.NETCore.ILAsm` gives me a binary (which is not marked executable by default) but it can not find `libcoreclr.so`

Basically.. what's the recommended way to actually install and use the IL Tools on a dotnet core install?

Ideally, there'd be `dotnet ildasm` and `dotnet ilasm`...

**Comment by karelz** [comment]
@RussKeldorph @russellhadley who knows about packaging ilasm? (I was surprised to see we ship it on NuGet)

**Comment by RussKeldorph** [comment]
The goal of #5214 was just to do the bare minimum required to separate these tools from the coreclr package but still make them available.  To answer the question: there is no recommended way to use them at this point.  Obviously, we should make sure they are marked executable, so that's an easy bug.  I'm not sure if or how they should be integrated into the dotnet/CLI experience, but my first guess would be that they should follow the same model as Roslyn.  I can follow up on this.

**Comment by RussKeldorph** [comment]
Let's see where dotnet/cli#6223 goes.

**Comment by karelz** [comment]
Should we close this issue as dupe of the one linked? IMO there's no point to have 2 tracking issues.
We can ask on the CLI issue to reopen this one if they reject it ... thoughts?

**Comment by ViktorHofer** [comment]
@karelz I think the other issue which is maintained by Peter is more than sufficient.

**Comment by karelz** [comment]
Closing ... @RussKeldorph feel free to reopen if you disagree.

**Comment by pjbgf** [comment]
@mletterle I have created a CLI version for ILDasm that is cross platform. If you try again `nuget install dotnet-ildasm` it should work now.  If you want to contribute or play around with the code, [here](https://github.com/pjbgf/) it is.
